### DISCUSSION TITLE
How to set window size?

### DISCUSSION THREAD

**Discussion by GreenSleeper** [folke/trouble.nvim#394] [Q&A]
I'm using `LazyVim` and I've enabled `trouble-v3`, but the window size configuration like before doesn't work now.
```lua
{
        "folke/trouble.nvim",
        branch = "dev",
        opts = {
            height = 30,
            width = 100,
        },
    }
```
How can set window size now?

**Comment by folke** [comment]
For splits, `opts.win.size` is normally just a number. Depending on the position of the split, it will be used as the height/width.

So you could already change this per mode like:

```lua
{
  modes = {
   diagnostics = {
    win = {size = 10}
  }
  }
}
```

Howver, I just pushed a change where for a split you can also define both a width/height.. Depending on the position one of the two will be used.

So ou can just do the below to chang ethe efaults
Use `opts.win.size = {width = 100, heigh = 30}`
